volume/volume.go

```diff
@@ -79,7 +79,6 @@ func (m *MountPoint) Setup() (string, error) {
 				return "", err
 			}
 			if runtime.GOOS != "windows" { // Windows does not have deprecation issues here
```

Comment:
@jhowardmsft Windows does not auto-create these directories, or is bind-mounting not supported at all?